### PR TITLE
Remove unnecessary filler values from flyteadmin sandbox config

### PR DESCRIPTION
**Pull request by katrogan**

### CODE REVIEW

File: deployment/sandbox/flyte_generated.yaml [hunk 1]
```diff
@@ -556,30 +556,11 @@ data:
...
         durationMinutes: 3
-    notifications:
-      type: local
-      region: "us-east-1"
-      publisher:
-        topicName: "foo"
-      processor:
-        queueName: "queue"
-        accountId: "bar"
     queues:
```
Comment (kumare3) [inline deployment/sandbox/flyte_generated.yaml]: how about these?

Comment (kumare3) [reply deployment/sandbox/flyte_generated.yaml]: queues i mean

Comment (katrogan) [reply deployment/sandbox/flyte_generated.yaml]: @kumare3 done

**Comment by wild-endeavor** [comment]
Which queues are available is a batch operator concern, but how namespaces map to queues is entirely admin.  If it's separated, won't we need some method of communication between them?

**Comment by katrogan** [comment]
@kumare3 any objections if I merge this and we save the queue configs for a separate ticket?

**Comment by kumare3** [comment]
> @kumare3 any objections if I merge this and we save the queue configs for a separate ticket?

@katrogan lets go ahead with it and have an open issue for this.

**Pull request merged**